tests/unit/ui/components/TestCarousel.js

```diff
@@ -0,0 +1,42 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import Carousel from 'ui/components/Carousel';
+
+
+describe(__filename, () => {
+  function shallowRender({
+    ...props
+  } = {}) {
+    return shallow(<Carousel {...props} />);
+  }
+
+  it('renders a Carousel', () => {
+    const root = shallowRender({ sections: [] });
+
+    expect(root).toHaveClassName('Carousel');
```

Comment:
Maybe you would like to assert for no `CarouselSection` components (since `sections` prop is empty).